images/linux/Ubuntu2004-README.md

```diff
@@ -173,12 +169,12 @@
 - 2.1.302 2.1.403 2.1.526 2.1.617 2.1.701 2.1.818 3.1.120 3.1.202 3.1.302 3.1.415 5.0.104 5.0.209 5.0.303 5.0.403
 
 ### Databases
-- MongoDB 5.0.4 (apt source repository: https://repo.mongodb.org/apt/ubuntu)
+- MongoDB 5.0.5 (apt source repository: https://repo.mongodb.org/apt/ubuntu)
 - PostgreSQL 14.1 (apt source repository: https://apt.postgresql.org/pub/repos/apt/)
 - sqlite3 3.31.1
 
 #### MySQL
-- MySQL 8.0.27
+- MySQL 8.0.26
```

Comment:
Expected version downgrade (see https://github.com/actions/virtual-environments/pull/4674 and https://github.com/actions/virtual-environments/pull/4685)